src/Artistic-1.0-cl8.xml

```diff
@@ -0,0 +1,142 @@
+<spdx name="Artistic License 1.0 w/clause 8" identifier="Artistic-1.0-cl8" osi-approved="true">
+  <urls>
+    <url>http://opensource.org/licenses/Artistic-1.0</url>
+  </urls>
+  <notes>This license was superseded by v2.0 This is Artistic License 1.0 as found on OSI site, including clause
+         8.</notes>
+  <license>
+    <title>
+      <p>The Artistic License</p>
+    </title>
+    <optional>
```

Comment:
this isn't optional here: http://git.spdx.org/?p=license-list.git;a=blob;f=Artistic-1.0-cl8.txt;h=5e0b00f4ed4838bb6f5b885fa3fdcb628d70c5e3;hb=HEAD

Let's make this not optional in github.